compiler/rustc_codegen_ssa/src/back/write.rs

```diff
@@ -533,7 +533,15 @@ fn copy_all_cgu_workproducts_to_incr_comp_cache_dir(
         if let Some(dwarf_object_file_path) = &module.dwarf_object {
             files.push(("dwo", dwarf_object_file_path.as_path()));
         }
-
+        if let Some(path) = &module.assembly {
+            files.push(("s", path.as_path()));
+        }
+        if let Some(path) = &module.llvm_ir {
+            files.push(("ll", path.as_path()));
+        }
+        if let Some(path) = &module.bytecode {
+            files.push(("bc", path.as_path()));
```

Comment:
Minor style nit:

```suggestion
            files.push((OutputType::Assembly.extension(), path.as_path()));
        }
        if let Some(path) = &module.llvm_ir {
            files.push((OutputType::LlvmAssemble.extension(), path.as_path()));
        }
        if let Some(path) = &module.bytecode {
            files.push((OutputType::Bitcode.extension(), path.as_path()));
```

I know that we currently don't use `OutputType::Object.extension()` for the `object_file_path` but well ^^